include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1894,6 +1894,11 @@ class MatrixFree : public Subscriptor
    * Categories run between the given values in the field
    * AdditionalData::cell_vectorization_category for non-hp-DoFHandler types
    * and return the active FE index in the hp-adaptive case.
+   *
+   * @note Following the behaviour of get_cell_category(), we return the
+   * maximum catergory of any cell batch. Only in the hp case, it is
+   * guaranteed that all cells and as a consquence all cell batches have
+   * the same categroy.
```

Comment:
There are a few typos, plus maybe something that can be clarified
```suggestion
   * maximum category of any cell batch. In the hp case, it is
   * guaranteed that all cells and as a consequence all cell batches in a range have
   * the same category. Otherwise, there may be different categories in different
   * cell batches.
```